core/DAPS/README.md

```diff
@@ -1,16 +1,212 @@
 # Dynamic Attribute Provisioning Service (DAPS)
 
+The infrastructure component "Digital Attribute Provisioning Service" (DAPS)
+ in a given IDS ecosystem enables enrichment of identities of organisations
+ and connectors with additional attributes.
+ 
+DAPS can be understood (like the [Certification Authority](../CA/README.md), too) as building block of the construct of an
+ [IDS Identity Provider](../../glossary/README.md#identity-provider). 
+
+These attributes are embedded by the DAPS into a [Dynamic Attribute Token (DAT)]().
+ The advantages of this technique of *dynamic attribute provisioning* instead
+ of embedding them into [X.509 certificates](https://en.wikipedia.org/wiki/X.509)
+ are:
+
+- Attribute revocation does not enforce certificate revocation and re-issuance.
+ This is also true if new attributes are added. 
+- By defining scopes or attribute sets, only the needed attributes can be
+ included in the DAT. This limits information leakage, so only needed
+ attributes are communicated.
+- Baseline certificates can be issued to enable connector deployment
+ in uncomplicated manner.
+- More complex scenarios can be created as soon as attributes are
+ assigned later on. 
+
 See also:
 - [Glossary "Dynamic Attribute Provisioning Service"](../../glossary/README.md#dynamic-attribute-provisioning-service)
 - Shortcut: `DAPS`
 
+---
+
+## Unique Identifier
+
+Each (TODO: COMPONENT/entry) connector in the IDS needs a valid, outlasting and unique
+ identifier, never be re-used for any other resource inside the IDS ecosystem.
+
+The architecture aims to be open to multiple [Certification Authorities](../CA/README.md)
+ (CAs) issuing certificates. This means, a truly unique identifier needs to consist of
+  the issuer of the certificate and the subject identifier. For an easy machine readable
+  identifier, two ´X.509v3´ extensions will be used:
+
+### Subject Key Identifier (SKI)
+### Authority Key Identifier (AKI)
+
+The concatenation of ´SKI´ and ´AKI´ provides a unique identifier valid - even if multiple
```

Comment:
What does this mean?

Reply:
Hm, bad explanation?!? Uniquness of identifier, even from different domain/sources aka CAs...

Reply:
'The concatenation of ´SKI´ and ´AKI´ provides a unique identifier valid - ...' sounds like there is something missing.

so it is meant like: 

'The concatenation of ´SKI´ and ´AKI´ provides an identifier which is unique - even if multiple CAs are able to issue valid certificates' ?